.github/workflows/validate-documentation.yml

```diff
@@ -16,7 +16,13 @@ jobs:
 
     steps:
     - name: check out code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v3.1.0
+
+    - name: verify links by lycheeverse
+      uses: lycheeverse/lychee-action@v1.5.1
+      with:
+        fail: true
+        args: "--threads 1 --max-concurrency 1 --verbose --no-progress './**/*.md' './**/*.html'"
```

Comment:
we could consider  removing `'./**/*.html'` and reverting the workflow name to `markdownlint` if that would lead to any problem.

_For me it could stay as it is 😄 _

Reply:
File filers are taken from default lychee configuration. There is no possibility to set other custom attributes and not setting file filters.

I would keep as it is.